### PR TITLE
Create Interface Malika

### LINKED ISSUE #5: Malika ismizga ismizga clas oching

### PR DESCRIPTION
**Pull request by Bakhtiyorovna**
Closes #5

### CODE REVIEW

File: pullrecuest/Malika.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace pullrecuest
+{
+    public interface Malika
+    {
+    }
+}
```
Comment (Yodgorovna) [inline pullrecuest/Malika.cs:6]: Saom Malikaxon: interface I bilan boshlanadi_

File: pullrecuest/Malika.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace pullrecuest
+{
+    public interface Malika
+    {
```
Comment (SamandarYigitaliyev) [inline pullrecuest/Malika.cs:4]: interface xato yaratilgan

**Review by SamandarYigitaliyev — CHANGES_REQUESTED** [review]

**Review by Yodgorovna — CHANGES_REQUESTED** [review]
Salom

File: pullrecuest/Malika.cs [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    }
```
Comment (rustambekovich) [inline pullrecuest/Malika.cs:5]: ha hamsi tushunarli

**Review by rustambekovich — APPROVED** [review]

**Review by SamandarYigitaliyev — APPROVED** [review]
Malatsy

**Review by Yodgorovna — APPROVED** [review]

**Review by Yodgorovna — APPROVED** [review]

**Pull request merged**